Gemfile

```diff
@@ -76,7 +76,7 @@ group :test do
   gem "shoulda-matchers", "~> 6.1"
   gem "simplecov"
   gem "simplecov-lcov"
-  gem "webdrivers"
+  gem "webdrivers", "= 5.3.0"
```

Comment:
@cdoyle-temple Is there supposed to be a space between the equal sign and the version number?